roles/powervs_client_enable_services/tasks/main.yml

```diff
@@ -51,6 +51,9 @@
       when:
         - client_config.squid.no_proxy_hosts is defined
         - not replace_noproxy_log.changed
+    - name: Check that you can connect (GET) to a page and it returns a status 200
```

Comment:
Test if internet is accessible and a 200 status is returned

Reply:
corrected